.dockerignore

```diff
@@ -0,0 +1,7 @@
+reset-database.sh
+node_modules
```

Comment:
good job here ensuring that docker does not end up copying the local node_modules into the container.